include/inviwo/core/properties/minmaxproperty.h

```diff
@@ -310,8 +308,10 @@ template <typename T>
 MinMaxProperty<T>& MinMaxProperty<T>::setRangeNormalized(const range_type& newRange) {
     dvec2 val = this->get();
 
-    val = (val - static_cast<double>(range_.value.x)) /
-          (static_cast<double>(range_.value.y) - static_cast<double>(range_.value.x));
+    const auto nomalizedValue =
```

Comment:
unused?

Reply:
ahh forgot to finish that one...

Reply:
could this be moved into the if statement?

Reply:
No since it depends on the old range...